boom! testing luminance-0.17.2 against nightly for nightly
boom! running: cargo +nightly build --frozen
boom! creating container for: cargo +nightly build --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly build --frozen cargobomb`
blam! e573b547e0a6c9623f78ab4ef7105d1ff40c3fa5f23ab15b0cc799a0a1d31764
boom! running `docker start -a e573b547e0a6c9623f78ab4ef7105d1ff40c3fa5f23ab15b0cc799a0a1d31764`
kablam!    Compiling luminance v0.17.2 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 3.3 secs
boom! running `docker rm -f e573b547e0a6c9623f78ab4ef7105d1ff40c3fa5f23ab15b0cc799a0a1d31764`
blam! e573b547e0a6c9623f78ab4ef7105d1ff40c3fa5f23ab15b0cc799a0a1d31764
boom! running: cargo +nightly test --frozen --no-run
boom! creating container for: cargo +nightly test --frozen --no-run
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly test --frozen --no-run cargobomb`
blam! 44c32916243da3495106d584f44362670cc1a89bd798d45fcd895615ebba5a21
boom! running `docker start -a 44c32916243da3495106d584f44362670cc1a89bd798d45fcd895615ebba5a21`
kablam!    Compiling luminance v0.17.2 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 3.14 secs
boom! running `docker rm -f 44c32916243da3495106d584f44362670cc1a89bd798d45fcd895615ebba5a21`
blam! 44c32916243da3495106d584f44362670cc1a89bd798d45fcd895615ebba5a21
boom! running: cargo +nightly test --frozen
boom! creating container for: cargo +nightly test --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly test --frozen cargobomb`
blam! cd72aaa99378b262aef475dceb7cd12754c367db75cfbd66534a01b123798f4f
boom! running `docker start -a cd72aaa99378b262aef475dceb7cd12754c367db75cfbd66534a01b123798f4f`
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 0.0 secs
kablam!      Running /target/debug/deps/luminance-156cd5d0fd754422
blam! 
blam! running 0 tests
blam! 
blam! test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured
blam! 
kablam!    Doc-tests luminance
blam! 
blam! running 8 tests
blam! test src/buffer.rs - buffer (line 26) ... FAILED
blam! test src/buffer.rs - buffer (line 34) ... FAILED
blam! test src/buffer.rs - buffer (line 41) ... FAILED
blam! test src/buffer.rs - buffer (line 8) ... FAILED
blam! test src/chain.rs - chain (line 30) ... FAILED
blam! test src/buffer.rs - buffer (line 50) ... FAILED
blam! test src/chain.rs - chain::Chain (line 18) ... FAILED
blam! test src/texture.rs - texture::Wrap::Repeat (line 89) ... FAILED
blam! 
blam! failures:
blam! 
blam! ---- src/buffer.rs - buffer (line 26) stdout ----
blam! 	error[E0425]: cannot find value `buffer` in this scope
blam!  --> <anon>:2:1
blam!   |
blam! 2 | buffer.clear(0.);
blam!   | ^^^^^^ not found in this scope
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /checkout/src/librustc/session/mod.rs:203
blam! note: Run with `RUST_BACKTRACE=1` for a backtrace.
blam! 
blam! ---- src/buffer.rs - buffer (line 34) stdout ----
blam! 	error[E0425]: cannot find value `buffer` in this scope
blam!  --> <anon>:2:1
blam!   |
blam! 2 | buffer.fill([1, 2, 3, 4, 5]);
blam!   | ^^^^^^ not found in this scope
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /checkout/src/librustc/session/mod.rs:203
blam! 
blam! ---- src/buffer.rs - buffer (line 41) stdout ----
blam! 	error[E0425]: cannot find value `buffer` in this scope
blam!  --> <anon>:2:1
blam!   |
blam! 2 | buffer.set(3, 3.14);
blam!   | ^^^^^^ not found in this scope
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /checkout/src/librustc/session/mod.rs:203
blam! 
blam! ---- src/buffer.rs - buffer (line 8) stdout ----
blam! 	error[E0412]: cannot find type `Buffer` in this scope
blam!  --> <anon>:2:13
blam!   |
blam! 2 | let buffer: Buffer<f32> = Buffer::new(5);
blam!   |             ^^^^^^ not found in this scope
blam! 
blam! error[E0433]: failed to resolve. Use of undeclared type or module `Buffer`
blam!  --> <anon>:2:27
blam!   |
blam! 2 | let buffer: Buffer<f32> = Buffer::new(5);
blam!   |                           ^^^^^^^^^^^ Use of undeclared type or module `Buffer`
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /checkout/src/librustc/session/mod.rs:203
blam! 
blam! ---- src/chain.rs - chain (line 30) stdout ----
blam! 	error: cannot find macro `chain!` in this scope
blam!  --> <anon>:3:12
blam!   |
blam! 3 | type Zoo = chain![i32, bool, f32]; // exactly the same type
blam!   |            ^^^^^
blam! 
blam! error[E0412]: cannot find type `Chain` in this scope
blam!  --> <anon>:2:12
blam!   |
blam! 2 | type Foo = Chain<i32,Chain<bool,f32>>;
blam!   |            ^^^^^ not found in this scope
blam!   |
blam!   = help: possible candidates are found in other modules, you can import them into scope:
blam!             `use std::io::Chain;`
blam!             `use std::iter::Chain;`
blam! 
blam! error[E0412]: cannot find type `Chain` in this scope
blam!  --> <anon>:2:22
blam!   |
blam! 2 | type Foo = Chain<i32,Chain<bool,f32>>;
blam!   |                      ^^^^^ not found in this scope
blam!   |
blam!   = help: possible candidates are found in other modules, you can import them into scope:
blam!             `use std::io::Chain;`
blam!             `use std::iter::Chain;`
blam! 
blam! error: aborting due to 3 previous errors
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /checkout/src/librustc_errors/lib.rs:455
blam! thread 'rustc' panicked at 'couldn't compile the test', /checkout/src/librustdoc/test.rs:270
blam! 
blam! ---- src/buffer.rs - buffer (line 50) stdout ----
blam! 	error[E0425]: cannot find value `buffer` in this scope
blam!  --> <anon>:3:17
blam!   |
blam! 3 | let all_elems = buffer.whole();
blam!   |                 ^^^^^^ not found in this scope
blam! 
blam! error[E0425]: cannot find value `buffer` in this scope
blam!  --> <anon>:7:12
blam!   |
blam! 7 | assert_eq!(buffer.at(3), Some(3.14));
blam!   |            ^^^^^^ not found in this scope
blam! 
blam! error[E0308]: mismatched types
blam!  --> <anon>:4:37
blam!   |
blam! 4 | assert_eq!(all_elems, vec![1, 2, 3, 3.14, 5]); // admit floating equalities
blam!   |                                     ^^^^ expected integral variable, found floating-point variable
blam!   |
blam!   = note: expected type `{integer}`
blam!              found type `{float}`
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /checkout/src/librustc/session/mod.rs:203
blam! 
blam! ---- src/chain.rs - chain::Chain (line 18) stdout ----
blam! 	error: cannot find macro `chain!` in this scope
blam!  --> <anon>:4:12
blam!   |
blam! 4 | type Zoo = chain![i32, bool, f32]; // Zoo == Foo
blam!   |            ^^^^^
blam! 
blam! error[E0412]: cannot find type `Chain` in this scope
blam!  --> <anon>:2:12
blam!   |
blam! 2 | type Foo = Chain<i32,Chain<bool,f32>>;
blam!   |            ^^^^^ not found in this scope
blam!   |
blam!   = help: possible candidates are found in other modules, you can import them into scope:
blam!             `use std::io::Chain;`
blam!             `use std::iter::Chain;`
blam! 
blam! error[E0412]: cannot find type `Chain` in this scope
blam!  --> <anon>:2:22
blam!   |
blam! 2 | type Foo = Chain<i32,Chain<bool,f32>>;
blam!   |                      ^^^^^ not found in this scope
blam!   |
blam!   = help: possible candidates are found in other modules, you can import them into scope:
blam!             `use std::io::Chain;`
blam!             `use std::iter::Chain;`
blam! 
blam! error[E0412]: cannot find type `Chain` in this scope
blam!  --> <anon>:3:12
blam!   |
blam! 3 | type Bar = Chain<Chain<i32, bool>, f32>;
blam!   |            ^^^^^ not found in this scope
blam!   |
blam!   = help: possible candidates are found in other modules, you can import them into scope:
blam!             `use std::io::Chain;`
blam!             `use std::iter::Chain;`
blam! 
blam! error[E0412]: cannot find type `Chain` in this scope
blam!  --> <anon>:3:18
blam!   |
blam! 3 | type Bar = Chain<Chain<i32, bool>, f32>;
blam!   |                  ^^^^^ not found in this scope
blam!   |
blam!   = help: possible candidates are found in other modules, you can import them into scope:
blam!             `use std::io::Chain;`
blam!             `use std::iter::Chain;`
blam! 
blam! error: aborting due to 5 previous errors
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /checkout/src/librustc_errors/lib.rs:455
blam! thread 'rustc' panicked at 'couldn't compile the test', /checkout/src/librustdoc/test.rs:270
blam! 
blam! ---- src/texture.rs - texture::Wrap::Repeat (line 89) stdout ----
blam! 	error[E0425]: cannot find value `coord_ith` in this scope
blam!  --> <anon>:3:1
blam!   |
blam! 3 | coord_ith = frac(coord_ith); // always between [0;1]
blam!   | ^^^^^^^^^ not found in this scope
blam! 
blam! error[E0425]: cannot find function `frac` in this scope
blam!  --> <anon>:3:13
blam!   |
blam! 3 | coord_ith = frac(coord_ith); // always between [0;1]
blam!   |             ^^^^ not found in this scope
blam! 
blam! error[E0425]: cannot find value `coord_ith` in this scope
blam!  --> <anon>:3:18
blam!   |
blam! 3 | coord_ith = frac(coord_ith); // always between [0;1]
blam!   |                  ^^^^^^^^^ not found in this scope
blam! 
blam! error: aborting due to previous error(s)
blam! 
blam! thread 'rustc' panicked at 'Box<Any>', /checkout/src/librustc/session/mod.rs:203
blam! 
blam! 
blam! failures:
blam!     src/buffer.rs - buffer (line 26)
blam!     src/buffer.rs - buffer (line 34)
blam!     src/buffer.rs - buffer (line 41)
blam!     src/buffer.rs - buffer (line 50)
blam!     src/buffer.rs - buffer (line 8)
blam!     src/chain.rs - chain (line 30)
blam!     src/chain.rs - chain::Chain (line 18)
blam!     src/texture.rs - texture::Wrap::Repeat (line 89)
blam! 
blam! test result: FAILED. 0 passed; 8 failed; 0 ignored; 0 measured
blam! 
kablam! error: test failed
boom! running `docker rm -f cd72aaa99378b262aef475dceb7cd12754c367db75cfbd66534a01b123798f4f`
blam! cd72aaa99378b262aef475dceb7cd12754c367db75cfbd66534a01b123798f4f
